onnxruntime/core/providers/cpu/nn/qlinearconv.cc

```diff
@@ -148,7 +148,7 @@ Status QLinearConv::Compute(OpKernelContext* context) const {
 #endif
 
   const auto* Xdata = X->template Data<uint8_t>();
-  const auto* Wdata = W->template Data<uint8_t>();
+  __attribute__((unused)) const auto* Wdata = W->template Data<uint8_t>();
```

Comment:
Oops, can you instead replace the expression below with Wdata? The __attribute__ syntax isn't portable to MSVC.

      GemmlowpMultiplyu8u8_u8(**W->template Data<uint8_t>()** + group_id * W_offset,